geoportailv3/static/js/measure/measuredirective.js

```diff
@@ -129,6 +130,8 @@ app.MeasureController = function($scope, $q, $http, ngeoDecorateInteraction,
   this.map_ = this['map'];
 
   var measureProfile = new ngeo.interaction.MeasureLength({
+    startMsg: goog.dom.getElement('measure-profile-start-message'),
```

Comment:
I'd create the elements in the JavaScript rather than relying on external ids, as this ties the directive to specific ids defined outside.